idd/Energy+.idd.in

```diff
@@ -397,6 +397,9 @@ Version,
   A1 ; \field Version Identifier
       \default ${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}
 
+FakeObject,
+  A1; \field Name
+
```

Comment:
Fake change. If everything worked well, I would expect to the see both the epJSON schema and the author warning for the embedded one, but the latter is missing

```diff
[  5%][1/19] Generating ../../Products/Energy+.schema.epJSON
Generating the epJSON schema!
- [ 10%][2/19] Generating EmbeddedEpJSONSchema.cc
-CMake Warning (dev) at /Users/julien/Software/Others/EnergyPlus/scripts/dev/generate_embeddable_epJSON_schema/generate_embedded_epJSON_schema.cmake:7 (message):
-  generating embedded epJSON schema
-This warning is for project developers.  Use -Wno-dev to suppress it.
```